settings.py

```diff
@@ -998,6 +998,8 @@
 DATADOG_API_KEY = None
 DATADOG_APP_KEY = None
 
+PROCESSOR_BACKEND = 'couch'  # either couch or postgres
```

Comment:
I think the full picture is very similar to what @czue did for the clean restore:
* domain level toggle to determine which backend to use
* setting to allow overriding toggle (`TEST_SHOULD_USE_SQL_BACKEND`)
* use `@override_settings(TEST_SHOULD_USE_SQL_BACKEND=xxx)` to run some tests with only one backend
* use decorator to run some tests with both backends: https://github.com/dimagi/commcare-hq/blob/multi-test-cleanup/corehq/ex-submodules/casexml/apps/phone/tests/restore_test_utils.py#L50-50